fml/trace_event.cc

```diff
@@ -65,8 +65,8 @@ void TraceSetTimelineMicrosSource(TimelineMicrosSource source) {
 }
 
 size_t TraceNonce() {
-  static std::atomic_size_t gLastItem;
-  return ++gLastItem;
+  static std::atomic_size_t last_item;
```

Comment:
Is there any convention for marking a variable as static or global?

Reply:
I think we left our comments within seconds of each other; see my other comment.